build.gradle

```diff
@@ -0,0 +1,292 @@
+buildscript {
+  repositories {
+    maven {
+      url 'https://plugins.gradle.org/m2/'
```

Comment:
Could repox be used?

Reply:
Yes, need to check if there is no issue when proxying the Gradle plugin registry in repox